.github/workflows/build-image.yml

```diff
@@ -0,0 +1,58 @@
+name: Build image
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_tag_name:
+        type: string
+        description: Specify the image tag name (Optional)
+      specify_commit:
+        type: string
+        description: Specify the commit hash (Optional)
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.specify_commit || github.ref }}
```

Comment:
Pardon me, if I enter `image_tag_name` will this checkout to the given tag branch ?

Reply:
Look like `image_tag_name` is only a prefix for tag name (?
@grieve54706  Is it possible to select a branch name and utilize its latest commit?

Reply:
I modified the field name to `docker_image_name` and removed the field `specify_commit`.
The `docker_image_name` is only for instead of the default docker image name.

Reply:
I think the `docker_image_name` seem not appropriate. I will modify it to `docker_image_tag_name`.